rusk/src/lib/chain.rs

```diff
@@ -55,19 +55,31 @@ impl RuskNode {
 /// Calculates the value that the coinbase notes should contain.
 ///
 /// 10% of the reward value goes to the Dusk address (rounded down).
-/// 80% of the reward value goes to the Block generator (rounded up).
+/// 70% of the reward value is considered fixed reward for Block Generator.
+/// 10% of the reward value is considered extra reward for Block Generator.
 /// 10% of the reward value goes to the all validators/voters of previous block
 /// (rounded down).
 const fn coinbase_value(
     block_height: u64,
     dusk_spent: u64,
-) -> (Dusk, Dusk, Dusk) {
+) -> (Dusk, Dusk, Dusk, Dusk) {
     let value = emission_amount(block_height) + dusk_spent;
```

Comment:
Can you rename this to something more meaningful, like `reward_value` or `total_value`?